UKHomeOffice/sgmr-service
# SPCR-298 Include pre-submitted, pre-cancelled and failed voyage plan statuses

## Description
The presubmitted, precanceled and failed statuses have been added in ManageReports. Unit test has also been created.

## To Test
- Go to manage voyage plans page
- You should see 3 additional new tabs
- You should be able to interact with them to see presubmitted, precanceled and failed reports (if you have any)

## Developer Checklist

\* Required






Ensure you delete the branch once the Pull Request is merged.
